app/controllers/admin/product_import_controller.rb

```diff
@@ -79,11 +79,9 @@ def check_spreadsheet_has_data(importer)
     end
 
     def save_uploaded_file(upload)
-      filename = 'import' + Time.zone.now.strftime('%d-%m-%Y-%H-%M-%S')
       extension = '.' + upload.original_filename.split('.').last
```

Comment:
You still have the dot in here and below in `import.`. Pau's suggestion would be apt now:
```suggestion
       extension = File.extname(upload.original_filename)
```

Reply:
Done :+1: